portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,17 @@ function addRandomGreeting() {
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function loadingcontent() {
```

Comment:
Can we give this a more descriptive name than "content"? For example, what content is it loading?

Reply:
Done, don't really have a specific content yet so just made it broad, could expand later if I have time.